include/swift/Frontend/Frontend.h

```diff
@@ -437,6 +437,16 @@ class CompilerInstance {
     return PrimarySourceFiles;
   }
 
+  /// Gets the Primary Source File if one exists, otherwise the main
```

Comment:
A list of conflicts crept into the *commit message* here. ("Sink a bunch of FrontendTool uses…")

Reply:
Removed them.